integration/nemo/examples/prompt_learning/prompt_learning.ipynb

```diff
@@ -0,0 +1,462 @@
+   "outputs": [],
+   "source": [
+    "script_name = \"prompt_learning_financial_phrase_bank_preprocessing.py\"\n",
+    "if not os.path.isfile(script_name):\n",
+    "    !wget -N \"https://raw.githubusercontent.com/NVIDIA/NeMo/main/scripts/dataset_processing/nlp/financial_phrase_bank/$script_name\"\n",
+    "else:\n",
+    "    print(f\"{script_name} already downloaded.\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dbff6b45",
+   "metadata": {},
+   "source": [
+    "#### 2. Download the Financial PhraseBank Dataset\n",
+    "\n",
+    "Download the `FinancialPhraseBank-v1.0.zip` dataset from [here](https://www.researchgate.net/profile/Pekka_Malo/publication/251231364_FinancialPhraseBank-v1.0/data/0c96051eee4fb1d56e000000/FinancialPhraseBank-v1.0.zip).\n",
+    "\n",
+    "Then extract it under `./data`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "860f8970",
+   "metadata": {},
+   "source": [
+    "#### 3. Preprocess the dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "47fac0b4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!python3 prompt_learning_financial_phrase_bank_preprocessing.py"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "56cd3a07",
+   "metadata": {},
+   "source": [
+    "#### 4. Split the dataset to simulate clients\n",
+    "Next, we use three clients to simulate federated learning for p-tuning with NeMo."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0ae8d5f2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!python3 data/split_financial_phrase_data.py --data_path data/FinancialPhraseBank-v1.0/financial_phrase_bank_train.jsonl --num_clients 3 --out_dir data/FinancialPhraseBank-v1.0_split"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f04052a7",
+   "metadata": {},
+   "source": [
+    "## Federated learning simulations\n",
+    "Next, we are using NVFlare's [simulator](https://nvflare.readthedocs.io/en/latest/user_guide/fl_simulator.html) to simulate each client training on their own dataset locally and all three clients training together using the [FedAvg](https://arxiv.org/abs/1602.05629) algorithm implemented in NVFlare.\n",
+    "\n",
+    "With this setting, we require a GPU with at least 16GB memory to run all clients in parallel on the same GPU. \n",
+    "If you have multiple GPUs in your system, you can use the `gpu` argument to assign one GPU for each client, e.g., `gpu=\"0,1\"`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6af89f8d",
+   "metadata": {},
+   "source": [
+    "#### 1. Local P-Tuning\n",
+    "First, we modify the configuration files to include the current directory path to access the dataset and pre-trained LLM."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1f59b4c0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!python3 modify_configs.py --job_folder \"jobs/gpt_p-tuning_local\""
```

Comment:
doesn't seem to have this script

Reply:
Can you pull again from this branch? The latest commit should have it.